tests/test_resources/upgrades/mpyl_config_upgraded.yml

```diff
@@ -0,0 +1,136 @@
+mpylVersion: 1.0.12
```

Comment:
Are these the MPyL release versions or will we have a separate versioning for the config structure?

Reply:
This supposed to follow the release semver scheme. With these intermediate versions it doesn't matter much what they are, but when for the multiple registries change we could target `1.3.0`